src/shared/components/input/input.component.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { FormattedMessage } from 'react-intl';
+import { useDebouncedCallback } from 'use-debounce';
+
+import { useDispatch } from 'react-redux';
+import { FilterType } from '../../../modules/filters/filters.types';
+import { setSearch } from '../../../modules/filters/filters.actions';
+import { Container, SearchIcon, InputWrapper, InputComponent, InputUnderline } from './input.styles';
+import messages from './input.messages';
+
+export const Input = () => {
+  const dispatch = useDispatch();
+
+  const debouncedOnTextChange = useDebouncedCallback((option: FilterType) => {
+    dispatch(setSearch(option));
+  }, 500);
+
+  return (
+    <Container>
+      <InputWrapper>
+        <FormattedMessage {...messages.placeholder}>
```

Comment:
Placeholder should be passed as a prop. 
Treat input as a dumb component which should be reused in other locations in different cases, for example - it should be also ready to be used for login and password. (In the future they want to add authorization here - maybe they will use this input)